components/eventing-controller/controllers/subscriptionv2/jetstream/test_utils.go

```diff
@@ -160,6 +172,19 @@ func startReconciler(ens *jetStreamTestEnsemble) *jetStreamTestEnsemble {
 	ens.K8sClient = k8sManager.GetClient()
 	require.NotNil(ens.T, ens.K8sClient)
 
+	// wait for the webhook server to get ready
```

Comment:
(nit) Can we make a test helper for startAndWaitWebhookServer(k8sManager), and use it in both eventMesh and jetStream? instead of duplicating the same code to start a webhook

Reply:
good point. will do